### DISCUSSION TITLE
compile for web?

### DISCUSSION THREAD

**Discussion by ehrenmann1977** [Budibase/budibase#5604] [General]
is it possible to compile the application as a web app ? where no need for docker, just a host and it will run locally from the web app. Any idea in this field?

**Comment by mike12345567** [comment]
Hi @ehrenmann1977 - this isn't something that Budibase is capable of, due to the fact it depends heavily on the backend applications for datasources, automations etc. The backend needs to depend on the database service (CouchDB), object stores (MinIO) and caching layers (Redis).
We are working on a single docker image install, which would be a lot easier to host for simple web apps, just run the single image and import your application to it.
